tests/unit/test_middleware/test_csrf_middleware.py

```diff
@@ -242,6 +242,28 @@ def post_handler2(data: dict = Body(media_type=RequestEncodingType.URL_ENCODED))
         assert response.json() == data
 
 
+def test_csrf_middleware_exclude_from_set_cookies() -> None:
+    @get("/protected-handler")
```

Comment:
```suggestion
def test_csrf_middleware_exclude_from_set_cookies() -> None:
    #  https://github.com/litestar-org/litestar/issues/3688
    #  middleware should be bypassed completely when excluded, so no cookies should be set
    
    @get("/protected-handler")
```

For future reference